cmd/lncli/cmd_open_channel.go

```diff
@@ -210,6 +210,11 @@ var openChannelCommand = cli.Command{
 				"propose to the remote peer (%q, %q)",
 				channelTypeTweakless, channelTypeAnchors),
 		},
+		cli.BoolFlag{
+			Name: "zero_conf",
+			Usage: "(optional) whether a zero-conf channel open " +
+				"should be attempted.",
+		},
```

Comment:
Similarly allow an scid alias to be passed along as well?

Reply:
Allowing a user to specify a scid alias is kind of dicey, what if they pass in one that's actually confirmed on-chain? Also they're randomly generated, not sure that they can really _prefer_ a certain scid